src/NationalInstruments.Analyzers/Properties/Resources.resx

```diff
@@ -345,4 +345,16 @@
   <data name="IdentifiersShouldBeSpelledCorrectlyTitle" xml:space="preserve">
     <value>Identifiers should be spelled correctly</value>
   </data>
+  <data name="NI1017_Description" xml:space="preserve">
+    <value>If you invoke a chain of methods with lambdas and that single line becomes very long, split each invocation onto its own line.</value>
```

Comment:
Maybe we should get rid of the "very long" verbiage given Jayson's comment elsewhere

Reply:
Done. I updated the comment to not talk about the length of the line.